python/doc/examples/reliability_sensitivity/reliability/plot_axial_stressed_beam.py

```diff
@@ -142,14 +142,14 @@
 
 # %%
 # We create a NearestPoint algorithm
-myCobyla = ot.Cobyla()
+myalgoOptim = ot.AbdoRackwitz()
```

Comment:
The "my" prefix is a bad habit we once had? Since you are renaming the variable, I would suggest "algoOptim" instead of "myalgoOptim"